script.js

```diff
@@ -1,4 +1,218 @@
+
+// function to generate random number
+function random(max) {
+  let randomNum = Math.floor(Math.random() * max) + 1;
+  return randomNum;
+}
+// create function to generate random item
+function randomOption() {
+  let randomItemNum = random(3);
+  let item;
+  if (randomItemNum == 1) {
+    item = 'scissors';
+  }
+  else if (randomItemNum == 2) {
+    item = 'paper';
+  }
+  else if (randomItemNum == 3) {
+    item = 'stone';
+  }
+  return item;
+}
+
+//**STAGE-2 Win-Loss Record */
+let winningCountComp = null;
+let winningCountPlayer = null;
+let drawCount = null;
+// **
+
+let recentWinner = 'yet to decide. keep playing.';
+
+// play normal SPS
+function playNormal(randomItem, input) {
+  let result;
+  if ((randomItem == 'scissors') && (input == 'paper')) {
```

Comment:
Lots of repeated code below. Can you use `||` statements in between your `&&` statements? E.g. if scissors and paper OR paper and stone OR stone and scissors: computer wins?

Repeating code is frowned upon in industry and you will lose points for it in interviews and take-home assignments, not to mention on the job.

Reply:
Thanks a lot for the feedback. The code in this PR was super redundant and ugly. Fixed the conditionals as well. Refactored the code and it shrunk drown to 138 lines from 218.  Perhaps something for me to come back to and check my improvements.

Reply:
Thanks for revising!